build.sh

```diff
@@ -55,24 +79,24 @@ BUILD_DIR=/build
 # Copy all packages, including any hidden dotfiles. Also copy the
 # local eal package and the lambda handler.
 ###
-cp -rT $VENV_DIR/lib/python3.8/site-packages/ $BUILD_DIR
-cp -rT $VENV_DIR/lib64/python3.8/site-packages/ $BUILD_DIR
-cp -r eal $BUILD_DIR
-cp lambda_handler.py $BUILD_DIR
+cp -rT "$VENV_DIR/lib/python$PY_VERSION/site-packages/" "$BUILD_DIR"
+cp -rT "$VENV_DIR/lib64/python$PY_VERSION/site-packages/" "$BUILD_DIR"
+cp -r eal "$BUILD_DIR"
+cp lambda_handler.py "$BUILD_DIR"
```

Comment:
I suggest using the long version of the command line flags here.

Reply:
Please see 7286f2d for these options in their full glory.